modules/model/src/main/scala/lucuma/itc/newTypes.scala

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc
+
+import cats.Order
+import cats.syntax.all.*
+import io.circe.Encoder
+import io.circe.Json
+import io.circe.JsonObject
+import io.circe.syntax.*
+import lucuma.core.data.Zipper
+import lucuma.core.data.ZipperCodec.given
+import lucuma.core.util.NewType
+import cats.data.NonEmptyChain
+
+private def customizedZipperEncoder[A: Encoder](
```

Comment:
Can this be shared with the instance in `client`?

Reply:
`model` is shared between client and server. But I'm actually removing this, it's no longer necessary.